### ISSUE TITLE
Naming of repo and app conflicts with fair use of trademark

### ISSUE THREAD

**Issue by nothingismagick** [tauri-wallet/tauri-wallet#1]
It is lovely to see this app, and the stuff you have built, however the explicit use of the protected wordmark of Tauri in the repo name and org name violates fair use. By using the UK and European Trademark of Tauri in this way, you are implicitly saying that this is an official project of the Tauri Community. Which it is not. 

Please rebrand.

**Comment by molin0000** [comment]
Haha, ok, I will change it.

**Comment by nothingismagick** [comment]
but also please change the org name

**Comment by nothingismagick** [comment]
i see you changed to aries wallet, maybe the org rename has to wait a minute or two.

**Comment by nothingismagick** [comment]
And really, it is truly flattering that you enjoy tauri so much that you would name your project after it. ❤️

**Comment by molin0000** [comment]
Yes, I like tauri very much. haha

**Comment by nothingismagick** [comment]
Also, don't forget this:

https://twitter.com/TauriApps/status/1517431288210333696

**Comment by molin0000** [comment]
I have changed all tauri-wallet to aries-wallet. I think aries is a friends of tauri haha...

**Comment by nothingismagick** [comment]
<img width="339" alt="image" src="https://user-images.githubusercontent.com/35242872/164675985-3fbcbc2f-9f00-4b7b-bc3a-dacd634771a4.png">

**Comment by molin0000** [comment]
> <img alt="image" width="339" src="https://user-images.githubusercontent.com/35242872/164675985-3fbcbc2f-9f00-4b7b-bc3a-dacd634771a4.png">

do you know how to change the left string?

**Comment by molin0000** [comment]
> <img alt="image" width="339" src="https://user-images.githubusercontent.com/35242872/164675985-3fbcbc2f-9f00-4b7b-bc3a-dacd634771a4.png">

I don't know how to modify it. so I created a new orgnization at https://github.com/aries-wallet  and I will remove this one later few days.

**Comment by nothingismagick** [comment]
Yeah, you can change the org name in its settings page, probably here:

https://github.com/organizations/tauri-wallet/settings/profile
